### DISCUSSION TITLE
Does it support other languages?

### DISCUSSION THREAD

**Discussion by Nirmal4G** [massCodeIO/snippet-collection#4] [Q&A] [answered]
The template only has `html` and `css` tag! But can we add JavaScript `js`, `C/C++`, `C#`, etc...? Basically what I'm asking is that this project is like a PasteBin for Code or Gist but with preview? If so, what are the technologies do you plan to support for Previews (_Outputs_) and what are the ones that are just snippets?

**Comment by antonreshetov** [comment, marked as answer]
@Nirmal4G For now, I think it will just be a collection of HTML & CSS. Other languages may be added in the future, but obviously without REPL

**Reply by Nirmal4G** [reply]
May be not REPL but a static output via web assembly without any input or simple inputs could be supported for other languages.

**Comment by Johann01** [comment]
What about TailwindCss?
